packages/devtools-source-map/src/tests/source-map.js

```diff
@@ -50,6 +50,11 @@ describe("source maps", () => {
       const urls = await setupBundleFixture("noroot");
       expect(urls).toEqual(["http://example.com/heart.js"]);
     });
+
+    test("Non-existing sourceRoot resolution with relative URLs", async () => {
+      const urls = await setupBundleFixture("noroot2");
+      expect(urls).toEqual(["http://example.com/heart.js"]);
+    });
```

Comment:
i'm not really sure what this test is documenting. It might be nice to see the webpack example in a test as well.

Reply:
The other tests cover webpack -- I initially wrote my patch a different way, which is how I knew that I needed to do this regexp thing.

Do you want some other change here as well?

Reply:
i see. that's okay.